# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1213947295 -7200
#      Fri Jun 20 09:34:55 2008 +0200
# Node ID 72c332d2db3c31950571999179026003bf16bc6e
# Parent  eb9ccb44ea41b08fc08987b10b76c7bd05c94680
add missing include in MatrixType.h

diff --git a/liboctave/ChangeLog b/liboctave/ChangeLog
--- a/liboctave/ChangeLog
+++ b/liboctave/ChangeLog
@@ -1,8 +1,12 @@
+2008-06-20  Jaroslav Hajek <highegg@gmail.com>
+
+	* MatrixType.h: Add missing include statement.
+
 2008-06-13  Michael Goffioul  <michael.goffioul@gmail.com>
 
 	* lo-mappers.cc (arg): Remove ambiguity about atan2(float,float) usage.
 	* fCmplxDET.cc (FloatComplexDET::initialize10): Likewise, for pow.
 
 2008-06-11  John W. Eaton  <jwe@octave.org>
 
 	* so-array.h, so-array.cc, Array-so.cc: Delete.
diff --git a/liboctave/MatrixType.h b/liboctave/MatrixType.h
--- a/liboctave/MatrixType.h
+++ b/liboctave/MatrixType.h
@@ -19,16 +19,18 @@ You should have received a copy of the G
 along with Octave; see the file COPYING.  If not, see
 <http://www.gnu.org/licenses/>.
 
 */
 
 #if !defined (octave_MatrixType_h)
 #define octave_MatrixType_h
 
+#include "oct-types.h"
+
 class Matrix;
 class ComplexMatrix;
 class FloatMatrix;
 class FloatComplexMatrix;
 class SparseMatrix;
 class SparseComplexMatrix;
 
 class
